python/tk_multi_loader/dialog.py

```diff
@@ -1322,7 +1430,8 @@ def _switch_profile_tab(self, new_index, combo_operation_mode):
             # history navigation, ask the currently visible
             # view to (background async) refresh its data
             model = self._entity_presets[self._current_entity_preset].model
-            model.async_refresh()
+            if isinstance(model, SgEntityModel):
+                model.async_refresh()
```

Comment:
I tried this approach of keeping the tab "type", but this proved less intuitive since the connection between a tab "type" and its model type had to be kept in mind and extra commented, not to mention harder to maintain if/when a model type changes.

I will add a comment to the current code.